agent/ae/ae.go

```diff
@@ -93,67 +93,64 @@ func (s *StateSyncer) Run() {
 
 FullSync:
```

Comment:
I think so since I've moved the tests as well but I'll check. Not sure how much they cover.